testing/internal/e2e/tests/base_plus/rate_limit_test.go

```diff
@@ -275,6 +275,14 @@ func TestCliRateLimit(t *testing.T) {
 
 	ctx := context.Background()
 
+	// TODO: Remove this whem CLI request time is improved (when cache daemon is enabled)
```

Comment:
```suggestion
	// TODO: Remove this when CLI request time is improved (when cache daemon is enabled)
```
FWIW, I'm not sure it'll get much better than it is right now on the LLB. Todd merged the fix that avoids starting the daemon if it's running, the rest is all keyring access.

Reply:
hm...